locales/templates/app.pot

```diff
@@ -300,17 +328,19 @@ msgstr ""
 #. Settings: strings displayed in the settings page,     Refer to UX spec 6.1:
 #. https://mozilla.invisionapp.com/share/24N3MMQB986#/310133927_6-1_Settings
 msgctxt "settings_list_capture"
-msgid "Enable screenshot service"
+msgid "Enable to take screenshots"
```

Comment:
Considering the technical limitation, we need to create a conceptual model that users must turn on the main switch to enable/customize the following screenshot features we provide, including taking screenshots with notification or FAB, which we refer to the "GO" button, and prompt a sorting panel right after users take a screenshot with the mentioned two ways. The attached shows that users need to turn on the main switch to find the other two options. 

As for copy, originally we decided to call the feature package "screenshot service," however, it might imply of something clouding. As a result, the copywriter and I decided to skip the term and only focused on the outcome of the switch itself. 

I'll add an explanation in the localization comment. Please let me know if this makes sense, thanks.

![image](https://user-images.githubusercontent.com/18019946/45408016-b3f37e00-b69d-11e8-82df-ad733abe65d9.png)

Reply:
Just to clarify my point: I'm just not sure "Enable to take screenshots" is the correct wording in English (and I don't really have an answer, I don't write copy and I'm not a native speaker). But I'd suggest to not get stuck on this, if we can't get anyone else to weight in.